Source/Core/Core/DSP/Jit/x64/DSPEmitter.cpp

```diff
@@ -128,6 +133,11 @@ bool DSPEmitter::FlagsNeeded() const
   return !(flags & Analyzer::CODE_START_OF_INST) || (flags & Analyzer::CODE_UPDATE_SR);
 }
 
+static void FallbackThunk(Interpreter::Interpreter& interpreter, UDSPInstruction inst)
+{
+  (interpreter.*Interpreter::GetOp(inst))(inst);
```

Comment:
I assume that deferring this lookup to execution time is a tradeoff for not having to worry about the compiler implementation of pointers to member functions...?

Reply:
Yes

Reply:
Oh, I know. The rest of the thunks should be free, or nearly so. I called this one out because of the added runtime cost from the lookup and the subsequent indirect branch. I understand it would take some extra work to avoid, though. It may not be worth the effort for the DSP JIT, but I was thinking it might be for the PPC JIT (assuming you plan to refactor the rest of Dolphin in this way).